docs/reference/modules/scripting/painless-syntax.asciidoc

```diff
@@ -207,7 +207,7 @@ In addition to Java's `enhanced for` loop, the `for in` syntax from groovy can a
 
 [source,painless]
 ---------------------------------------------------------
-for (item : list) {
+for (item in list) {
```

Comment:
I don't think this works, I just tried in 6.2. Have you tested this syntax in 5.2 where you opened this issue? Can you post a working example if so?